dist/components/Blob.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from "react";
+
+import styled from "styled-components";
+
+import * as THREE from "three";
+import SimplexNoise from "simplex-noise";
+
+const Blob = () => {
+  useEffect(() => {
+    let $canvas = document.querySelector("#blob canvas"),
+      canvas: any = $canvas,
+      renderer = new THREE.WebGLRenderer({
+        canvas: canvas,
+        context: canvas.getContext("webgl2"),
+        antialias: true,
+        alpha: true
+      }),
+      simplex = new SimplexNoise();
+
+    renderer.setSize(500, 400);
+    renderer.setPixelRatio(window.devicePixelRatio || 1);
+
+    let scene = new THREE.Scene(),
+      camera = new THREE.PerspectiveCamera(30, 5 / 4, 0.1, 500);
+
+    camera.position.z = 5;
+
+    let geometry = new THREE.SphereGeometry(0.8, 128, 128);
+
+    let material = new THREE.MeshPhongMaterial({
+      color: 0xff8379,
+      shininess: 40
+    });
+
+    let lightTop = new THREE.DirectionalLight(0xffffff, 0.55);
+    lightTop.position.set(0, 500, 200);
+    lightTop.castShadow = true;
+    scene.add(lightTop);
+
+    let lightBottom = new THREE.DirectionalLight(0xfeda22, 0.8);
+    lightBottom.position.set(0, -500, 400);
+    lightBottom.castShadow = true;
+    scene.add(lightBottom);
+
+    let ambientLight = new THREE.AmbientLight(0x86dcea);
+    scene.add(ambientLight);
+
+    let sphere: any = new THREE.Mesh(geometry, material);
```

Comment:
You can use https://github.com/react-spring/react-three-fiber for types and avoid using `any` in this file